src/sth_helper.js

```diff
@@ -140,15 +140,43 @@
     return sthConfig.OPERATION_TYPE_PREFIX + request.method.toUpperCase();
   }
 
-  module.exports = function (aSthConfig) {
+  /**
```

Comment:
No, no, you are right ;) There are 2 getISODateString() functions, one in sth_logger.js (not exported) and one in sth_helper.js (the one exported). I did it this way since sth_helper.js depends on sth_logger.js (to log :) ) and to avoid making sth_logger.js depend on sth_helper.js, which obviously it not possible.

We have 2 options as far as my understanding here: 1) Leave it as it is :p and 2) include a third file (sth_helper_no_log.js or something like that) to include and to export the mentioned getISODateString() function. Both sth_helper.js and sth_logger.js would depend on (and take as an argument when requiring it) this new file (contents).

What do you think? Any preference on your side :)